code/modules/surgery/prosthetic_replacement.dm

```diff
@@ -17,6 +17,7 @@
 
 /datum/surgery_step/add_prosthetic
 	name = "add prosthetic"
+	accept_any_item = TRUE
 	implements = list(/obj/item/bodypart = 100, /obj/item/organ_storage = 100, /obj/item/chainsaw = 100, /obj/item/melee/synthetic_arm_blade = 100)
```

Comment:
Since this is depreciated, you might want to change this to the same list as cavity implant.

```suggestion
	implements = list(/obj/item = 100)
```

Not that it really matters if the comments about the accept_any_item var in the parent are true, and it does ignore this, but if it's not working this would be the first thing to try and fix it.